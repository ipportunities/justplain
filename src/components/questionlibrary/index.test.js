import React from 'react'
import { shallow } from 'enzyme'
import QuestionLibrary from './index'

describe('Question Library', () => {
  it('should render correctly', () => {
    const component = shallow(<QuestionLibrary />)
    expect(component).toMatchSnapshot()
  })
})
