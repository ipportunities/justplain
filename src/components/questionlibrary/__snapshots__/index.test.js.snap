// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question Library should render correctly 1`] = `
<div>
  <Menu />
  <Switch>
    <Route
      exact={true}
      path="/question-library"
    >
      <QuestionLibraryOverview />
    </Route>
    <Route
      path="/question-library/edit/:id"
    >
      <QuestionLibraryEdit />
    </Route>
  </Switch>
</div>
`;
